test_scripts/API/VehicleData/common.lua

```diff
@@ -455,17 +473,83 @@ function m.unexpectedDisconnect(pParam)
   utils.wait(1000)
 end
 
+--[[ @unexpectedDisconnect: Unexpected disconnect sequence for two apps for different <vd_param> parameters
+--! @parameters:
+--! pParam1: name of the VD parameter for 1st app
+--! pParam2: name of the VD parameter for 2st app
+--! @return: none
+--]]
+function m.unexpectedDisconnectTwoApps(pParam1, pParam2)
+  m.getHMIConnection():ExpectNotification("BasicCommunication.OnAppUnregistered", { unexpectedDisconnect = true })
+  :Times(2)
+  m.getHMIConnection():ExpectRequest("VehicleInfo.UnsubscribeVehicleData")
+  :ValidIf(function(_, data)
+    if data.params[pParam1] == true or data.params[pParam2] == true then
+      return true
+    else
+      return false, "VehicleInfo.UnsubscribeVehicleData request contains unexpected parameter" ..
+        utils.tableToString(data.params)
+    end
+  end)
+  :Times(2)
+  actions.mobile.disconnect()
+  utils.wait(1000)
+end
```

Comment:
Can we update existing `unexpectedDisconnect()` function as we did for `ignitionOff()`?